board.rb

```diff
@@ -0,0 +1,58 @@
+require 'pry'
+
+class Board
+  include Mover
+  attr_reader :board, :key_array
+  def initialize
+    @board = {}
+    @key_array = []
+    board_hash
+  end
+
+  def board_maker
+    (0..7).each do |x|
+      (0..7).each do |y|
+        key_array << [x,y]
+      end
+    end
+  end
+
+  def board_hash
+    board_maker
```

Comment:
Weird for one method to pass data to the other through instance variables. If U want board_hash to use data from board_maker just have board_maker return an array. Like

`board_maker.each do |num|`

But name board_maker better.